src/status_im/ui/screens/syncing/sheets/enter_password/views.cljs

```diff
@@ -0,0 +1,88 @@
+(ns status-im.ui.screens.syncing.sheets.enter-password.views
+  (:require [quo.react-native :as rn]
+            [status-im.ui.screens.syncing.sheets.enter-password.styles :as styles]
+            [re-frame.core :as re-frame]
+            [quo.core :as quo]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.toolbar :as toolbar]
+            [clojure.string :as string]
+            [taoensso.timbre :as log]
+            [status-im.utils.security :as security]
+            [status-im.ethereum.core :as ethereum]
+            [status-im.utils.fx :as fx]
+            [status-im.native-module.core :as status]
+            [quo2.components.info.information-box :as information-box]
+            [status-im.ui.components.qr-code-viewer.views :as qr-code-viewer]
+            [status-im.utils.handlers :refer [>evt]]))
+
+(def entered-password (atom ""))
+(def connection-string  (atom ""))
+
+(defn qr-code-view-with-connection-string [connection-string]
+  (log/info "connection-string ===> " connection-string)
+  (let [window-width       @(re-frame/subscribe [:dimensions/window-width])
+        valid-cs?          (string/starts-with? connection-string "cs")]
+    [:<>
+     (if valid-cs?
+       [rn/view {:style styles/body-container}
+        [rn/text {:style styles/header-text} "Sync code generated"]
```

Comment:
"Sync code generated" - should be an i18n string?

Reply:
yes definitely, shall move to i18n